pyasic/config/mining.py

```diff
@@ -442,6 +451,15 @@ def from_epic(cls, web_conf: dict):
                         ),
                         algo=TunerAlgo.voltage_optimizer(),
                     )
+                elif algo_info.get("BoardTune") is not None:
+                    return cls.hashrate_tuning(
+                        hashrate=algo_info["BoardTune"].get("Target"),
+                        throttle_limit=algo_info["BoardTune"].get(
```

Comment:
Bit confused about these, is this just a temperature aware throttle?

If so, it should go in as part of the `power_scaling` config stuff, https://github.com/UpstreamData/pyasic/blob/master/pyasic/config/power_scaling.py

It may need to be renamed to something else, not sure if power scaling is completely accurate, it was originally based on braiins' old dynamic power scaling, which was only power, but now everyone seems to have hashrate tuning so scaling, so maybe the name should be changed to performance scaling or something...